tests/ballerina-integration-test/src/test/resources/grpc/clients/grpc_secured_unary_client.bal

```diff
@@ -34,7 +34,6 @@ function testUnarySecuredBlocking() returns (string) {
                 versions: ["TLSv1.2","TLSv1.1"]
             },
             ciphers:["TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA"],
-            sslVerifyClient:"require",
```

Comment:
Why are we removing this?

Reply:
Recently "sslVerifyClient" has been removed from secureSocket record. Since the record was an open/unsealed record, it allows to have additional fields. Therefore secureSocket did not give any compilation error. But now we have made it to a closed and it cannot have such additional fields in the record. That is the reason for removing it.

Reply:
@Manuri We don't need to have 'sslVerifyClient' for client side. We only need that in the server side to enable 2-way SSL.